doc/Language/101-basics.pod6

```diff
@@ -325,13 +339,13 @@ have both won two matches. That simple sort doesn't account for the number of
 sets won, which is the secondary criterion to decide who has won the
 tournament.
 
-=head3 X<C<stable sort>>
+=head2 X<Stable sort|stable sort (Basics)>
```

Comment:
You are adding (basics) all over, or only those that have another entry somewhere? Because I don't remember this to be indexed. Also, not so sure it should be indexed...